f5_openstack_agent/tests/functional/esd/test_esd_migrating.py

```diff
@@ -14,14 +14,15 @@
 #   limitations under the License.
 
 from copy import deepcopy
-from f5_openstack_agent.lbaasv2.drivers.bigip.icontrol_driver import \
-    iControlDriver
 import json
 import logging
 import os
+
```

Comment:
This, too, breaks convention:
```python
import json
import logging
import os
import pytest
import requests

from filepath import FilePath

from ....lbaasv2.drivers.bigip.icontrol_driver import iConstrolDriver

from ..testlib.bigip_client import BigIpClient
from ..testlib.fake_rp import FakeRPCPlugin
from ..testlib.service_reader import LoadbalancerReader
```
This makes us closer to how OpenStack community handles imports consistently and makes F5 look better.

Reply:
I think 'filepath', 'requests', and 'pytest' need to be in a section, they are external dependencies.

There need to be _at least_ 3 sections.

One for builtins/stdlib
One for external/pip installed
One for our local code (all relative).

Reply:
I'm fine with sub dividing our own imports into, e.g. app code and test code, if that seems better to you.

Reply:
We should be following OpenStack style guidelines (hacking): https://docs.openstack.org/hacking/latest/user/hacking.html#imports